clash-prelude/src/Clash/Class/BitPack/BitIndex.hs

```diff
@@ -0,0 +1,148 @@
+{-|
+Copyright  :  (C) 2013-2016, University of Twente
+License    :  BSD2 (see the file LICENSE)
+Maintainer :  Christiaan Baaij <christiaan.baaij@gmail.com>
```

Comment:
* Update the copyright notice
  * Save @christiaanb the burden of being the sole maintainer :wink: